### DISCUSSION TITLE
Machine translation for Spanish?

### DISCUSSION THREAD

**Discussion by arissont** [dfint/.github#7] [Q&A] [answered]
Good morning, I was going to ask you if you couldn't translate automatically with some translators like google or bing in transifex and then I can review it personally or the Hispanic community for this great game

**Comment by insolor** [comment, marked as answer]
In general, mass machine translation is a very bad idea. We had such thing for Russian 11 years ago, when we had just started the project. And it took several years to fix such translations. I think, it would be better, if we just had them translated manually.

For Spanish, it is even worse: there are a lot of translations made by people, still not reviewed. And if we add 10 thousand machine translated strings, you'll never tell apart which strings are translated by people and which are machine translated, until you stumble upon such strings in game.

So, I strongly advise against such translation, and for sure I'm not doing this for any language, which I don't curate explicitly. And even for languages which I curate, I won't do it, never again.

**Reply by insolor** [reply]
Also, there is a rather big community for Spanish translation, for sure I won't do such things only by request of one person.

**Reply by arissont** [reply]
I get your point and it makes all the sense in the world it would be to mess up people's translations.
The problem is that there are people who make grammar mistakes. I'm just a simple fan. I have a discord group, which is the only one in the Hispanic dwarf fortress community. If you can promote it for several, coordinate in the same translation and thus translate the best possible quality. And nothing more to say, we value your effort in this project.

**Reply by arissont** [reply]
I will leave a discord link https://discord.gg/rmBNXr2RcR

**Reply by insolor** [reply]
@arissont please contact with people in the following forum branch http://www.bay12forums.com/smf/index.php?topic=156549.new#new

I have a lot of things to do, for the localization project, and apart from it. I won't do coordination for languages other than Russian, Ukrainian, Belarusian. I have only one head, unfortunately, I cannot do coordination for all the communities.
